api/authentication/service.md

```diff
@@ -113,6 +113,8 @@ For any strategy allowed in `authStratgies`, a user can call `app.service('/auth
 - Create a JWT for the entity returned by the strategy
 - Return the JWT (`accessToken`) and the additional information from the strategy
 
+> For `local` strategy, the user has to be created before doing auth, otheriwse, a 401 `NotAuthenticated` error will be send.
```

Comment:
Ah.  This is a very important note.  Thank you.   Could you please fix the spelling of "otherwise"?

Reply:
fixed, thanks. This is the when-grammarly-stops-working-moment. 😂